docs/automated-testing/shadow-testing/README.md

```diff
@@ -54,7 +54,7 @@ Some advantages of shadow testing are:
 - We can test real-life scenarios with real-life data.
 - We can simulate scale with replicated production traffic.
 
-## References  
+## References
```

Comment:
Hi @TessFerrandez ! Great to see testing space changes are being made in this playbook. 

As part of the CNCF landscape, and inspired from the limitations with Diffy(mentioned in the tools above), we built keploy, completely open-source - https://github.com/keploy/keploy. Recently, we got a lot of users from Nutanix(using keploy in Nutanix open-source repos as well), Suzuki, Amazon, Reliance shadow testing using keploy to record and replay live traffic along with virtualised infra. 

It'll be really awesome, if you can add keploy as part of the [list above](https://github.com/microsoft/code-with-engineering-playbook/pull/1062/files#diff-d31def673572208f93e34b755be8c7e6f7db0fb66773b0294d8dba1e27b0691eR35), so that we can help developers looking to solve the shadow testing problem. Happy to answers any questions you may have. :)

Reply:
Sure thing... please add a PR, and add me as a reviewer if you want

Reply:
Awesome! I just added a PR for the same. Please review :) 

https://github.com/microsoft/code-with-engineering-playbook/pull/1063